client/src/features/signup/SignUp.tsx

```diff
@@ -39,50 +87,58 @@ const SignUp = () => {
       <LeftPart />
       <section className="flex justify-center my-auto h-9/12 w-1/2 tablet_l_max:w-full">
         <div className="flex flex-col justify-center h-full w-7/12 mobile_xl_max:w-9/12">
-          <h2 className="flex-initial text-4xl mb-2 mobile_l_max:text-3xl">
+          <h2 className="flex-initial text-4xl mb-3 mobile_l_max:text-3xl">
             Create an Account
           </h2>
-          <p className="flex-initial mb-12">
-            Already have an account?{' '}
+          <div className="mb-12 flex mobile_m_max:flex-col">
+            <p className="flex-initial mr-1 mobile_l_max:text-sm">
+              Already have an account?
+            </p>
             <Link to="/login" className="underline font-bold">
               Login
             </Link>
-          </p>
+          </div>
           <form action="" className="flex flex-col w-full">
             <label htmlFor="email" className="text-lg mb-2">
               Email Address
             </label>
             <div className="relative">
               <input
+                id="email"
                 type="email"
-                className="mb-8 h-10 text-xl w-full pl-8"
-                onChange={(e) =>
-                  setInputs({ ...inputs, email: e.target.value })
-                }
+                className="h-10 text-xl w-full pl-8 rounded"
+                onChange={(e) => handleOnChange(e)}
+                onBlur={(e) => handleOnBlur(e)}
```

Comment:
```suggestion
                onChange={handleOnChange}
                onBlur={handleOnBlur}
```
We can make it shorter like this. The same goes for line 130-131.